graphql/schemabuilder/pagination.go

```diff
@@ -514,22 +514,22 @@ func (c *connectionContext) applyTextFilter(ctx context.Context, nodes []interfa
 
 	g, ctx := errgroup.WithContext(ctx)
 	matchStrings := filter.GetMatchStrings(*args.FilterText)
-
-	expensiveNodesToKeep, err := c.applyTextFilterNotBatchedExpensive(ctx, nodes, matchStrings, filterTextFieldsNotBatchedExpensive)
-	if err != nil {
-		return nil, err
-	}
-
-	nodesToKeep, err := c.applyTextFilterNotBatched(ctx, nodes, matchStrings, filterTextFieldsNotBatched)
-	if err != nil {
-		return nil, err
-	}
-
-	batchedNodesToKeep, err := c.applyBatchTextFilter(ctx, nodes, matchStrings, filterTextFieldsBatched)
-	if err != nil {
-		return nil, err
-	}
-
+	var nodesToKeep []bool
+	var expensiveNodesToKeep []bool
+	var batchedNodesToKeep []bool
+	var err error = nil
```

Comment:
I think it's a race condition to have three goroutines updating the same error variable?  (Also you don't need `= nil` here because the zero-value of `error` is `nil`)